setup.py

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env python
 import sys
-from setuptools import setup
+from setuptools import setup, Distribution
 from setuptools.command.test import test as TestCommand
 from sys import platform
 from platform import architecture
-import shutil
 
 if platform == 'win32' and architecture()[0] == '32bit':
-    shutil.copy2('win/sndfile32.dll', 'win/sndfile.dll')
-    sndfile = [('', ['win/sndfile.dll', 'win/sndfile_license'])]
+    sndfile = [('lib', ['win32/libsndfile.dll', 'win32/sndfile_license'])]
```

Comment:
why not keep the DLL install directory as before (i.e. `''`)? If you don't use a subdirectory, you don't need to do the path manipulation abomination in `dlopen()`.